ArduPlane/Parameters.cpp

```diff
@@ -267,28 +267,35 @@ const AP_Param::Info Plane::var_info[] = {
     
     // @Param: STALL_PREVENTION
     // @DisplayName: Enable stall prevention
-    // @Description: Enables roll limits at low airspeed in roll limiting flight modes. Roll limits based on aerodynamic load factor in turns and scale on ARSPD_FBW_MIN that must be set correctly. Without airspeed sensor, uses synthetic airspeed from wind speed estimate that may both be inaccurate.
+    // @Description: Enables roll limits at low airspeed in roll limiting flight modes. Roll limits based on aerodynamic load factor in turns and scale on AIRSPEED_MIN that must be set correctly. Without airspeed sensor, uses synthetic airspeed from wind speed estimate that may both be inaccurate.
     // @Values: 0:Disabled,1:Enabled
     // @User: Standard
     ASCALAR(stall_prevention, "STALL_PREVENTION",  1),
 
-    // @Param: ARSPD_FBW_MIN
+    // @Param: AIRSPEED_CRUISE
```

Comment:
I guess  AIRSPEED_CRUISE has nothing to do with CRUISE mode right?  So it's really a target airspeed.  In Copter we use WPNAV_SPEED and in Rover it's WP_SPEED.  So it's a feature of the Auto navigation.. nothing to do with the sensor.

Reply:
Cruise airspeed is the standard aviation term for this value

Reply:
It's your baby so I won't say any more after this, but I think the airspeed is really more like the value's type.  We don't prefix our various parameters that hold altitudes with ALT_ or BARO_.  The feature should go at the front of the parameter name.